docs/samples/Microsoft.ML.Samples/Dynamic/SsaSpikeDetectorTransform.cs

```diff
@@ -0,0 +1,96 @@
+    // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site. 
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using Microsoft.ML.Runtime.Data;
+    using Microsoft.ML.Runtime.Api;
+    using Microsoft.ML.Runtime.TimeSeriesProcessing;
```

Comment:
make indent consistent with other files, 8 spaces

Reply:
actually that is needed anymore, since we are aiming at using the entire file. The other samples will get corrected as well.

Reply:
Actually no need to indent anymore, since we are using the entire file. 

---
In reply to: [232375443](https://github.com/dotnet/machinelearning/pull/1578#discussion_r232375443) [](ancestors = 232375443)